frontend/front/src/pages/Public/Components/CardsSection.js

```diff
@@ -41,27 +41,43 @@ function CardsSection({
           alt={title}
           src={mediaSource}
           sx={{
-            backgroundSize: "contain",
-            height: { xs: "200px", sm: "300px", md: "300px", lg: "300px" },
+            objectFit: "cover",
+            display: "block",
+            margin: "auto",
+            width: "100%",
             minWidth: "500px",
             maxWidth: { xs: "353px", lg: "500px" },
+            height: { xs: "281px", md: "281px", lg: "281px" }, // 16:9 ratio
           }}
         />
       );
     } else if (mediaType === "iframe") {
       return (
         <Box
-          component="iframe"
-          title={title}
-          src={mediaSource}
           sx={{
+            display: "flex",
+            justifyContent: "center",
```

Comment:
This setup blocks part of Youtube videos on smaller screens - almost half the video on iPhone SE. I think we're better off with explicit sizing at the breakpoints since iframes aren't responsive. I tried this to good results:
```
<Box
          sx={{
            display: "flex",
            justifyContent: "center",
            alignItems: "center",
            width: "100%",
            minWidth: { xs: "353px", smm: "425px", sm: "500px" },
            maxWidth: { xs: "353px", smm: "425px", sm: "500px" },
            height: { xs: "198px", smm: "239px", sm: "281px" }, // 16:9 ratio
            overflow: "hidden",
            margin: "auto",
          }}
        >
          <Box
            component="iframe"
            title={title}
            src={mediaSource}
            sx={{
              width: { xs: "353px", smm: "425px", sm: "500px" },
              // minWidth: "100%",
              height: "100%",
              border: "none",
            }}
          />
        </Box>
```

Reply:
Fixed it